assertk-jvm/src/main/kotlin/assertk/assertions/throwable.kt

```diff
@@ -0,0 +1,78 @@
+@file:JvmName("ThrowableJVMKt")
+
+package assertk.assertions
+
+import assertk.Assert
+
+/**
+ * Returns an assert on the throwable's stack trace.
+ */
+fun <T : Throwable> Assert<T>.stackTrace() = prop("stack trace", { it.stackTrace.map { it.toString() } })
+
+/**
+ * Asserts the throwable's cause is an instance of the expected java class.
+ * @see [hasCauseWithClass]
+ * @see [hasRootCauseInstanceOf]
+ */
+@Deprecated(
+    message = "Use cause().isNotNull { it.isInstanceOf(jclass) } instead.",
```

Comment:
If we're going to be bumping major with the release, should we keep deprecated APIs?

Reply:
I have plans for those, but imo should be a separate commit. This one is big enough